static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -299,13 +302,21 @@ const buildPageTargetting = (
     // This can be removed once we get sign-off from third parties who prefer to use appNexusPageTargeting.
     page.pageAdTargeting = pageTargeting;
 
-    return pageTargeting;
+    return Promise.resolve(pageTargeting);
```

Comment:
because this is an async function, it automatically returns a promise. therefore you don't need to wrap `pageTargeting` here